editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -1201,8 +1201,34 @@ export function parseCode(
         forEachLeft(fromDeclaration, (exportDeclaration) => {
           pushArbitraryNode(exportDeclaration)
         })
-        // Handle imports.
+      } else if (
+        TS.isClassDeclaration(topLevelElement) &&
+        topLevelElement.modifiers != null &&
+        topLevelElement.modifiers.some((modifier) => modifier.kind === TS.SyntaxKind.ExportKeyword)
+      ) {
+        // Handle classes.
+        // Check if this is the 'default' export, which is to say the entire export for the file.
```

Comment:
warning, helpful comment is not completely right: in the ESModules (this is how our ts code is compiled too), a module can have named exports -and- default exports simultaneously. The default export is attached under the name `default`...